extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -359,6 +359,13 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     });
   }
 
+  /** @override */
+  cryptoUnavailable(bytes) {
```

Comment:
Why do you need to pass the bytes?  Seems like you're looking for a boolean indicator of what to do on crypto being unavailable but your having a signature header?  If so, how about this in AmpA4A:

allowAMPRenderIfSignatureWithoutCrypto() {
     return false;
}

Doubleclick then just returns if experiment feature is enabled.

Reply:
done